modules/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/step11_rgmb_homo.md

```diff
@@ -0,0 +1,88 @@
+# Reactor Geometry Mesh Builder Example: Homogeneous Sodium-Cooled Fast Reactor Core (ABTR)
+
+This example illustrates the use of RGMB mesh generators to define a 3D hexagonal geometry core with homogeneous assemblies (ABTR ([!cite](shemon2015abtr)), constructed earlier in this tutorial using base mesh generators).
+
+!media tutorial04_meshing/rgmb_abtr_stepbystep.png
+       id=tutorial04-rgmb_abtr_stepbystep
+       caption=Visualization of meshing steps to build the 3D ABTR core with RGMB mesh generators.
+       style=width:75%;display:block;margin-left:auto;margin-right:auto;
+
+## ReactorMeshParams
+
+[ReactorMeshParams.md] contains global mesh/geometry parameters including whether the final mesh is 2D or 3D, Cartesian or hexagonal, assembly pitch, and the axial discretization for the final extruded geometry. This information will be accessible to the other RGMB mesh generators and consistently used.
+
+!listing reactor_examples/rgmb_abtr/rgmb_abtr.i
+         id=tutorial04-rgmb_abtr-rmp
+         caption=ABTR RGMB Reactor Mesh Parameters example.
+         block=Mesh/rmp
+
+## PinMeshGenerator
+
+This example does not have any pin-level geometry as the assemblies are homogenized. However, we still start with [PinMeshGenerator.md]. To define homogenized assemblies, [PinMeshGenerator.md] is used with `use_as_assembly` set to `true`. This tells [CoreMeshGenerator.md] to treat this object as an assembly for stitching into the core lattice. In addition, `homogenized` = `true` is used to indicate that this region is homogenized and [SimpleHexagonGenerator.md] should be called to discretize the assembly instead of [PolygonConcentricCircleMeshGenerator.md].
```

Comment:
```suggestion
This example does not have any pin-level geometry as the assemblies are homogenized. However, we still start with [PinMeshGenerator.md]. To define homogenized assemblies, [PinMeshGenerator.md] is used with [!param](/PinMeshGenerator/use_as_assembly) set to `true`. This tells [CoreMeshGenerator.md] to treat this object as an assembly for stitching into the core lattice. In addition, setting [!param](/PinMeshGenerator/homogenized) to `true` is used to indicate that this region is homogenized and [SimpleHexagonGenerator.md] should be called to discretize the assembly instead of [PolygonConcentricCircleMeshGenerator.md].
```

Reply:
same for all other parameters below, this will help maintain the tutorial up to date